packages/ra-ui-materialui/src/field/ArrayField.stories.tsx

```diff
@@ -0,0 +1,162 @@
+import * as React from 'react';
+import { MemoryRouter } from 'react-router';
+import {
+    RecordContextProvider,
+    useListContext,
+    useRecordContext,
+} from 'ra-core';
+import { Card, ThemeProvider, createTheme } from '@mui/material';
+
+import { ArrayField } from './ArrayField';
+import { Datagrid, SingleFieldList } from '../list';
+import { ChipField } from './ChipField';
+import { SimpleShowLayout } from '../detail';
+import { TextField } from './TextField';
+import { Pagination } from '../list/pagination';
+
+export default { title: 'ra-ui-materialui/fields/ArrayField' };
+
+let books = [
+    { id: 1, title: 'War and Peace', author_id: 1 },
+    { id: 2, title: 'Les Misérables', author_id: 2 },
+    { id: 3, title: 'Anna Karenina', author_id: 1 },
+    { id: 4, title: 'The Count of Monte Cristo', author_id: 3 },
+    { id: 5, title: 'Resurrection', author_id: 1 },
+];
+
+export const Basic = () => (
+    <MemoryRouter>
+        <ArrayField record={{ id: 123, books }} source="books">
+            <SingleFieldList>
+                <ChipField source="title" />
+            </SingleFieldList>
+        </ArrayField>
+    </MemoryRouter>
+);
+
+export const PerPage = () => (
```

Comment:
This one would benefit from setting up the I18nProvider

Reply:
You're right, fixed!